data_scientist_skills/data_engineering.py

```diff
@@ -76,3 +80,106 @@ def process_salary_estimate(dataframe, only_mean = True):
     temp_df.drop(columns = 'salary_estimate', inplace = True)
 
     return temp_df
+
+
+###Sector Encoding Section
+
+def make_sector_group_column(df, give_group_profiles = False):
```

Comment:
The second parameter isn't used actually/giving group profiles isn't implemented here. I have the code but I thought it better to leave it out unless desired